release.sh

```diff
@@ -0,0 +1,48 @@
+set -o errexit
+
+VERSION="$(date +%Y.%m.%d)"
+LASTEST_VERSION="$(git tag --list | tail -n 1)"
+
+if [ "${VERSION}" = "${LASTEST_VERSION}" ]; then
+    echo "Nothing to release"
+    exit 0
+fi
+
+# Make sure local fork is updated
+git fetch -p --all
+git checkout master
+git merge upstream/master
```

Comment:
How about `git merge --ff-only upstream/master`? This'll help prevent the creation of a merge commit in case something is weird about the branch structure.